### PR TITLE
Upgrade to pulumi-terraform-bridge v2.3.0

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: sdk/python/pulumi_digitalocean/tables.py [hunk 1]
```diff
@@ -147,9 +142,6 @@
     "serviceSubnet": "service_subnet",
     "sha1Fingerprint": "sha1_fingerprint",
     "snapshotId": "snapshot_id",
-    "spacesAccessId": "spaces_access_id",
-    "spacesEndpoint": "spaces_endpoint",
-    "spacesSecretKey": "spaces_secret_key",
```
Comment (lukehoban) [inline sdk/python/pulumi_digitalocean/tables.py:152]: @pgavlin Just to check - are these removals expected as a result of the codegen changes?

File: sdk/nodejs/certificate.ts [hunk 2]
```diff
@@ -6,17 +6,6 @@ import * as utilities from "./utilities";
 
 import {CertificateType} from "./index";
 
-/**
- * Provides a DigitalOcean Certificate resource that allows you to manage
- * certificates for configuring TLS termination in Load Balancers.
- * Certificates created with this resource can be referenced in your
- * Load Balancer configuration via their ID. The certificate can either
- * be a custom one provided by you or automatically generated one with
- * Let's Encrypt.
```
Comment (lukehoban) [inline sdk/nodejs/certificate.ts:15]: @stack72 @pgavlin Do we know why we lost this (and one other) summary comment?

File: sdk/python/pulumi_digitalocean/tables.py (hunk 1 above)
Comment (pgavlin) [reply sdk/python/pulumi_digitalocean/tables.py:152]: There shouldn't be any codegen changes with this payload unless DO was on an older version of the bridge than I was aware of.

Comment (pgavlin) [reply sdk/python/pulumi_digitalocean/tables.py:152]: *SDK codegen changes

Comment (pgavlin) [reply sdk/python/pulumi_digitalocean/tables.py:152]: Okay, it looks like this has something to do with provider configuration. All of the removed properties are top-level entities in the config.

**Pull request merged**